docs/source/user-guide/general/metrics_guide.rst

```diff
@@ -0,0 +1,190 @@
+.. _metrics_guide:
+
+====================
+Metrics Guide
+====================
+
+In this guide we'll explain how to customize the metrics that deepchecks uses to validate and monitor your model.
+Controlling the metrics helps you shape the checks and suites according to the specifics of you use case.
+
+**Structure:**
+
+* `Default metrics <#default-metrics>`__
+* `Alternative metrics <#alternative-metrics>`__
+* `List of Supported Strings <#list-of-supported-strings>`__
+* `Custom Metrics <#custom-metrics>`__
+
+
+Default metrics
+===============
+All of the checks that evaluate model performance, such as SingleDatasetPerformance, come with
+default metrics.
+
+The default scorers by task type are:
+
+Tabular
+_______
+
+Classification, binary or multiclass averaged over the classes:
+
+*   Accuracy
+*   Precision
+*   Recall
+
+Multiclass classification per class:
+
+*   F1
+*   Precision
+*   Recall
+
+Regression:
+
+*   Negative RMSE (negative RMSE so that the convention that greater is better is kept)
+*   Negative MAE
+*   R2
+
+Vision
+______
+
+Classification:
+
+*   Precision
+*   Recall
+
+Object detection:
+
+*   Mean average precision
+*   Mean average recall
+
+To run a check with the default metrics, run it without passing any value to the "scorer" parameter. For example:
+
+:doc:`ClassPerformance </checks_gallery/vision/model_evaluation/plot_class_performance>`:
+
+.. code-block:: python
+
+    check = ClassPerformance()
+    result = check.run(train_ds, test_ds, mnist_model)
+
+
+Alternative metrics
+===================
+Sometimes the defaults don't fit the specifics of the use-case.
+If this is the case, you can pass a list of supported metric strings or a dict in the format {metric name: metric} as a
+parameter to the check.
+
+The metrics in the dict can be existing:
```

Comment:
```suggestion
The ``metric`` in the dict can be any of the following:
```

Reply:
@shir22 I disagree because after the list there's "or your own implementation"

Reply:
The original wording doesn't sound valid (=and I'm not sure if it is grammatically correct.
Feel free to change to something else if you don't like the suggested one.